example/__tests__/asyncstorage.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { commands as cmd } from './commands';
+import testCases from '../examples/tests';
+
+describe('Async Storage functional tests', () => {
+  it('should be visible', async () => {
+    // wait until content loads, as android's waitForSelectorTimeout setting does not seem to work
+    await new Promise((r) => setTimeout(r, 3000));
+    const el = await cmd.elementByLabel('functional-view');
+    await expect(await el.isExisting()).toEqual(true);
+  });
+
+  const testNames = Object.keys(testCases);
+  describe('storing / reading values', () => {
+    for (const name of testNames) {
+      it(`${name}`, async () => {
```

Comment:
```suggestion
      it(name, async () => {
```

Reply:
This was done due to jest eslint rule. I removed it now.